pennylane/transforms/decompositions/two_qubit_unitary.py

```diff
@@ -326,37 +302,42 @@ def _decomposition_3_cnots(U, wires):
 
 
 def two_qubit_decomposition(U, wires):
-    r"""Recover the decomposition of a two-qubit unitary :math:`U` in terms of
-    elementary operations.
-
-    The work of `Shende, Markov, and Bullock (2003)
-    <https://arxiv.org/abs/quant-ph/0308033>`__ presents a fixed-form
-    decomposition of :math:`U` in terms of single-qubit gates and
-    CNOTs. Multiple such decompositions are possible (by choosing two of ``{RX,
-    RY, RZ}``). Here we choose the ``RY``, ``RZ`` case (fig. 2 in the above) to
-    match with the default decomposition of the single-qubit :class:`~.Rot` operations
-    as ``RZ RY RZ``. The most general form of the decomposition is:
-
-    .. figure:: ../../_static/two_qubit_decomposition.svg
+    r"""Decompose a two-qubit unitary :math:`U` in terms of elementary operations.
+
+    It is known that an arbitrary two-qubit operation can be implemented using a
+    maximum of 3 CNOTs. This transform first determines the required number of
+    CNOTs, then decomposes the operator into a circuit with a fixed form.  These
+    decompositions are based a number of works by Shende, Markov, and Bullock
+    `(1) <https://arxiv.org/abs/quant-ph/0308033>`__, `(2)
+    <https://arxiv.org/abs/quant-ph/0308045v3>`__, `(3)
```

Comment:
Minor, but `[1], [2], [3]` feels more consistent to me given the QML website!

Reply:
Actually when I did that, I got some warnings about references when building the docs!

Reply:
Ah. Better to leave it as-is, then to worry about fixing Sphinx warnings